tests/observability/metrics/test_general_metrics.py

```diff
@@ -106,10 +106,11 @@ def test_vm_name_in_virt_launcher_label(self, fedora_vm_without_name_in_label):
 class TestVirtHCOSingleStackIpv6:
     @pytest.mark.ipv6
     @pytest.mark.polarion("CNV-11740")
-    @pytest.mark.s390x
     def test_metric_kubevirt_hco_single_stack_ipv6(self, prometheus, ipv6_single_stack_cluster):
+        if not ipv6_single_stack_cluster:
```

Comment:
please use tests.network.connectivity.conftest.fail_if_not_ipv6_supported_cluster
we do not use programatic skips